projects/make-a-vis/src/lib/visualization-view/main/main.component.ts

```diff
@@ -46,6 +47,10 @@ export class MainComponent {
       }))
     ).subscribe(visualizations => {
       this.visualizations = visualizations;
+      this.visualizations.forEach((vis) => {
+        vis.data.shortDescriptionText = vis.data.description.length <= 50 ?
```

Comment:
slicing can be done in the template

Reply:
removed it, now slicing is being done from template